explorer/interpreter/value.h

```diff
@@ -344,10 +344,12 @@ class NominalClassValue : public Value {
   auto base() const -> std::optional<Nonnull<const NominalClassValue*>> {
     return base_;
   }
+  auto vtable() const -> std::optional<Nonnull<const VTable*>> { return vptr_; }
```

Comment:
What if we said that every class has a vtable, but it's empty if the class has no virtual methods? That seems like it could simplify a lot of the optional-handling in this PR.

Reply:
Agree, I considered it, but though it might an memory overhead when unused. But less runtime overhead I guess then. Happy to have that simplification.